components/sub-components/SelectedArticle.jsx

```diff
@@ -0,0 +1,66 @@
+import { IoChatbubbleOutline } from "react-icons/io5";
+import { FaRegHeart } from "react-icons/fa";
+import styles from "./SelectedArticle.module.css";
+import { lineSpinner } from "ldrs";
+import { useState, useEffect } from "react";
+import { fetchArticleById } from "../../utils/api.js";
+import { Link } from "react-router-dom";
+
+lineSpinner.register();
+
+const SelectedArticle = ({article_id, votes, loading, setLoading}) => {
+  
+  const [selectedArticle, setSelectedArticle] = useState([]);
+  
+  useEffect(() => {
+    setLoading(true);
+    fetchArticleById(article_id).then((fetchedArticle) => {
+      setLoading(false);
+      setSelectedArticle(fetchedArticle);
+    });
+  }, []);
+
+  return loading ? (
+    <div className={styles.loading_symbol}>
+      <l-line-spinner
+        size="40"
+        stroke="3"
+        speed="1"
+        color="black"
+      ></l-line-spinner>
+    </div>
+  ) : (
+    <div className={styles.article_card}>
+      <div className={styles.author}>{selectedArticle.author}</div>
+      <div className={styles.created_at}>
+        {new Date(selectedArticle.created_at).toLocaleDateString()}
+      </div>
+      <div className={styles.text}>
+        <div className={styles.title}>
+          <h4>
+            <b>{selectedArticle.title}</b>
+          </h4>
+        </div>
```

Comment:
Really good ternary for the loading state, the only thing I can pick on here is that you've got a lot of `<div>` tags, I think it would be great to skim through all your div's and see if you could replace them with a HTML tag that is a bit more specialised to the role

For example line 35 the div is showing text, so it could be a `h2`, `h3`, `h4`, or `p` tag etc.